js/components/portal-dashboard/header-menu.tsx

```diff
@@ -101,7 +93,7 @@ export class HeaderMenuContainer extends React.PureComponent<IProps, IState> {
         {items && items.map((item: any, i: number) => {
```

Comment:
```suggestion
        {items && items.map((item, i) => {
```
I think if you leave off the types they should be inferred properly.

Reply:
Fixed.